cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1421,6 +1439,69 @@ def test_create_non_deprecated_problems(self):
         template_display_names = [template['display_name'] for template in templates[0]['templates']]
         self.assertEqual(template_display_names, ['Annotation', 'Poll', 'Survey'])
 
+    def test_deprecated_no_advance_component_button_with_admin_settings(self):
```

Comment:
I do not see a need for this test. `test_cannot_create_deprecated_problems()` already asserts that if all of the XBlock types in the "Advanced Modules List" are disabled, the button does not show.